### ISSUE TITLE
[BUG] unexpected change in behavior of npm view --json of invalid version of a package

### ISSUE THREAD

**Issue by MylesBorins** [npm/cli#1639]
### Current Behavior:
```bash
$ npm view --json lodash@0.0.0
undefined
$
```

### Expected Behavior:
In previous versions of npm there would be no content returned
```bash
$ npm view --json lodash@0.0.0
$
```

### Steps To Reproduce:
Run above commands

### Environment:
OSX 10.15.6 
Node.js v14.6.0
npm v7.0.0-beta.2

**Comment by darcyclarke** [comment]
This should be addressed when we land: https://github.com/npm/cli/pull/1657

**Comment by MylesBorins** [comment]
This has not been entirely fixed

npm v6.x output
```bash
[~]$ npm view --json lodash@0.0.0
[~]$
```

npm 7.x output
```bash
[~]$ npm view --json lodash@0.0.0

[~]$
```

Notice the additional newline that we have now. This is probably a non issue for most cases but it did break some internals on CITGM (but were patched easily enough)

**Comment by ruyadorno** [comment]
landed in `v7.0.0-beta.11`
